.github/workflows/test_conan.yml

```diff
@@ -0,0 +1,74 @@
+name: Tests (Conan)
+
+on:
+  pull_request:
+    branches: [main]
+  workflow_call:
+
+  # Allows you to run this workflow manually from the Actions tab
+  workflow_dispatch:
+
+jobs:
+  run_tests:
+    name: Conan build
+    runs-on: ubuntu-24.04
```

Comment:
this one runs on ubuntu 24.04 and release.yml uses this workflow and runs on ubuntu 22.04
probably their 'runs-on' should be equal?

Reply:
No, release.yml should release sc-machine built in minimal maintainable ubuntu - ubuntu-22.04